test-suite/src/test/scala/org/scalajs/testsuite/javalib/ListTest.scala

```diff
@@ -320,6 +320,23 @@ trait ListTest[F <: ListFactory] extends CollectionTest[F] {
       expect(al1.get(1)).toEqual("zero")
       expect(al1.get(2)).toEqual("five")
 
+      println(al)
+      val al1Iter = al1.listIterator
+      expect(al1Iter.hasNext).toBeTruthy
+      expect(al1Iter.next()).toEqual("nine")
+      expect(al1Iter.hasNext).toBeTruthy
+      expect(al1Iter.next()).toEqual("zero")
+      expect(al1Iter.hasNext).toBeTruthy
+      expect(al1Iter.next()).toEqual("five")
+      expect(al1Iter.hasNext).toBeFalsy
+      expect(al1Iter.hasPrevious).toBeTruthy
+      expect(al1Iter.previous()).toEqual("five")
+      expect(al1Iter.hasPrevious).toBeTruthy
+      expect(al1Iter.previous()).toEqual("zero")
+      expect(al1Iter.hasPrevious).toBeTruthy
+      expect(al1Iter.previous()).toEqual("nine")
+      expect(al1Iter.hasPrevious).toBeFalsy
```

Comment:
Why did this fail before the fix? It seems that all of this never changes sizes.